### DISCUSSION TITLE
How change display date format of data picker / range?

### DISCUSSION THREAD

**Discussion by akaytatsu** [cloudscape-design/components#1417] [General]
In components:

Date Input: https://cloudscape.design/components/date-input/?tabId=playground
Date Picker: https://cloudscape.design/components/date-picker/?tabId=playground
Date Range Picker: https://cloudscape.design/components/date-range-picker/?tabId=playground
The default display date format is yyyy/mm/dd.

How can I change this?

**Comment by rubencarvalho** [comment]
Hello Thiago, 

Our components currently don't support changing the display date format and we don't have short-term plans to implement this.

We did take a note for your request and will use it as a data point when planning future date-related components enhancements.

Thank you,
Ruben

**Comment by wysez** [comment]
I second this request. Can/often is a very odd UX when typical date input follow MM/DD/YYYY.

**Reply by just-boris** [reply]
Your proposed format is ambiguous for people across the world. 09/03/2023 may be interpreted either as March, 9th or September, 3rd.

To build inclusive interfaces for people across the world, you need to avoid ambiguous patterns

**Reply by IanKabitzVF** [reply]
I agree that a universal standard for an unambiguous date format is a great idea. Unfortunately my users didn't seem to respond well to the argument, and want the date format to reflect their locality (DD/MM/YYYY for UK). Therefore I cannot use this component at all. I would urge the team to reconsider making this configurable.

**Comment by just-boris** [comment]
As per this comment https://github.com/cloudscape-design/components/discussions/1417#discussioncomment-7092620, we are not planning to make format configurable

**Comment by MwSpaceLLC** [comment]
Please, follow: https://github.com/cloudscape-design/components/issues/2741
